test/extensions/extensions_build_system.bzl

```diff
@@ -8,7 +9,12 @@ def envoy_extension_cc_test(
         name,
         extension_name,
         **kwargs):
-    if not extension_name in EXTENSIONS:
+    extensions = select({
+        "//bazel:windows_x86_64": envoy_all_extensions(WINDOWS_SKIP_TARGETS),
+        "//bazel:linux_ppc": envoy_all_extensions(PPC_SKIP_TARGETS),
+        "//conditions:default": envoy_all_extensions(),
+    })
+    if not extension_name in extensions:
```

Comment:
This, unfortunately, isn't compatible with how `select()` works today.
The call to `select()` doesn't immediately evaluate to a value, that decision only happens when the "SelectObject" returned by `select()` is used as a parameter to a rule. https://github.com/bazelbuild/bazel/issues/8419 might provide some more insights to the issue.

Reply:
That's our understanding as well. Guidance/workarounds are appreciated, that's why we've shared this non-working PR. It ties into https://github.com/envoyproxy/envoy/issues/7903